modules/sildisco/src/IdPDisco.php

```diff
@@ -96,7 +99,7 @@ public function handleRequest(): void
         $metadata = MetaDataStorageHandler::getMetadataHandler();
         $sp = $metadata->getMetaData($spEntityId, 'saml20-sp-remote');
 
-        $t = new Template($this->config, 'selectidp-links', 'disco');
+        $t = new Template($this->config, 'selectidp-links');
```

Comment:
Poor disco. No longer wanted. :)

Reply:
So sad. Another missed piece from the SSP 2.0 upgrade. That argument was `$defaultDictionary`. We only have one dictionary.